paradox-menu.el

```diff
@@ -505,6 +508,22 @@ fetching the list.")
 	  (setq paradox--current-filter "Starred")
 	  (paradox-sort-by-package nil))))
 
+(defun paradox-filter-regex ()
+  "Show only packages matching regexp"
+  (interactive)
+  (let* ((regex (read-regexp "Enter regexp: "))
```

Comment:
Could you make this an interactive argument for the function? (Do you know how?)